internal/app/secretless/listeners/ssh/listener.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/x509"
 	"encoding/pem"
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1